lua/gatekeeper_config.lua

```diff
@@ -22,10 +22,13 @@ function gatekeeper_init()
 
 	if gatekeeper_server == true then
 		local gkf = require("gk")
-		local gk_conf, ggu_lcore = gkf(net_conf, numa_table)
+		local gk_conf, ggu_lcore, sol_lcore = gkf(net_conf, numa_table)
```

Comment:
OK, although this forces the declaration of `n_lcores` out of `lua/gk.lua` and into `lua/gatekeeper_config.lua`.